### DISCUSSION TITLE
Ability to include a "product type" as sales condition.

### DISCUSSION THREAD

**Discussion by outline4** [craftcms/commerce#2359] [Ideas]
### Description

Hi, at the moment we have the following conditions to match a sale: 

Start- Enddate  
User Groups  
Product Variant  
Categories  

There **really** should be a "Product Type" as well. 

- Craft Commerce version: 2.2.8

**Comment by lukeholder** [comment]
Will happen once we introduce a custom condition builder for purchasables to hook into. For now you will need to use categories.

**Reply by amphibian** [reply]
Any update on this feature? Still not an option in Commerce 4. It feels hacky to assign categories to products for the sole purpose of sale targeting.

**Reply by MoritzLost** [reply]
@lukeholder I was just looking for this feature and found this issue. We want to apply a specific discount to one product type only. At the moment we can only target specific products, but not the product type. I agree that creating categories just for that is hacky, and it requires adding the category to every product manually.

Are there plans to add a `Product type` condition to the new condition builder?
